### PR TITLE
Adds LogLevel to RunConfig on mobile

### PR DESCRIPTION
**Pull request by carsonfarmer**
What else, if anything, is needed to make this accessible outside the SDK?

### CODE REVIEW

File: mobile/mobile.go [hunk 1]
```diff
@@ -121,7 +122,8 @@ func MigrateRepo(config *MigrateConfig) error {
 // Create a gomobile compatible wrapper around Textile
 func NewTextile(config *RunConfig, messenger Messenger) (*Mobile, error) {
 	node, err := core.NewTextile(core.RunConfig{
-		RepoPath: config.RepoPath,
+		RepoPath:  config.RepoPath,
+		LogLevels: map[string]string{"*": config.LogLevel},
```
Comment (sanderpick) [inline mobile/mobile.go]: I think need more control here... gonna be super noisy. Could you allow something like `config.LogLevels` plural, as JSON (name: level)

Comment (carsonfarmer) [reply mobile/mobile.go]: Yup for sure, that's what we have on desktop etc.

**Comment by carsonfarmer** [comment]
Needs tests to be updated, so still in wip. Ignore for now pls.

**Comment by carsonfarmer** [comment]
*Warning: This PR requires modifications to the RN bridge before it will work nicely.*

File: mobile/mobile.go [hunk 2]
```diff
@@ -120,18 +121,17 @@ func MigrateRepo(config *MigrateConfig) error {
 
 // Create a gomobile compatible wrapper around Textile
 func NewTextile(config *RunConfig, messenger Messenger) (*Mobile, error) {
+	var logLevels map[string]string
+	if config.LogLevels != "" {
+		err := json.Unmarshal([]byte(config.LogLevels), logLevels)
```
Comment (sanderpick) [inline mobile/mobile.go]: I think `&logLevels` here

Comment (carsonfarmer) [reply mobile/mobile.go]: 👍

**Comment by sanderpick** [comment]
Can we wrap this one up soon-ish? Would be nice to hand-off log level control to textile-mobile.

**Comment by carsonfarmer** [comment]
I removed the WIP and merged master.

**Comment by carsonfarmer** [comment]
This will need to be passed to `MobileNewTextile` to keep our current behavior:

```
'{
  "tex-broadcast":   "DEBUG",
  "tex-core":        "DEBUG",
  "tex-datastore":   "DEBUG",
  "tex-ipfs":        "DEBUG",
  "tex-mill":        "DEBUG",
  "tex-repo":        "DEBUG",
  "tex-repo-config": "DEBUG",
  "tex-service":     "DEBUG",
}'
```

**Comment by carsonfarmer** [comment]
GTG @sanderpick

**Pull request merged**